airbyte-ci/connectors/base_images/base_images/sanity_checks.py

```diff
@@ -146,24 +146,6 @@ async def check_user_can_read_dir(container: dagger.Container, user: str, dir_pa
         raise errors.SanityCheckError(f"{dir_path} is not readable by {user}.")
 
 
-async def check_user_cant_write_dir(container: dagger.Container, user: str, dir_path: str):
```

Comment:
Dagger started outputing exit code 1, but not throwing the exception as our code assumed. Removing the whole check because I don't see how this is useful.